exercises/routing/exercise/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "angular2-webpack-build",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "postinstall": "rimraf typings & typings install",
+    "tsc": "tsc",
+    "typings": "typings",
+    "clean": "rimraf dist",
+    "webpack": "webpack",
+    "build": "npm run clean && cross-env NODE_ENV=development webpack",
+    "start": "cross-env NODE_ENV=development webpack-dev-server --content-base dist"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "repository": {},
+  "devDependencies": {
+    "awesome-typescript-loader": "2.2.1",
+    "cross-env": "2.0.1",
+    "css-loader": "0.24.0",
+    "css-to-string-loader": "0.1.1",
+    "html-webpack-plugin": "2.22.0",
+    "json-loader": "0.5.4",
+    "postcss-cssnext": "^2.4.0",
+    "postcss-import": "8.0.2",
+    "postcss-loader": "^0.9.1",
+    "raw-loader": "0.5.1",
+    "rimraf": "2.5.4",
+    "to-string-loader": "^1.1.3",
+    "typescript": "2.0.2",
+    "typings": "1.3.2",
+    "webpack": "2.1.0-beta.25",
+    "webpack-dev-server": "2.1.0-beta.6"
+  },
+  "dependencies": {
+    "@angular/common": "2.0.1",
+    "@angular/compiler": "2.0.1",
+    "@angular/core": "2.0.1",
+    "@angular/forms": "2.0.1",
+    "@angular/platform-browser": "2.0.1",
+    "@angular/platform-browser-dynamic": "2.0.1",
+    "@angular/router": "3.0.0",
+    "basscss": "^8.0.1",
```

Comment:
same comment as from David in pull request #24 (David: I think we shouldn't be using any "advanced" css framework, it might confuse people specially since we are not explaining css in the training)